### DISCUSSION TITLE
Auto increase of node size

### DISCUSSION THREAD

**Discussion by ArthurViret** [mermaid-js/mermaid#4942] [Ideas]
Hello teams, 
It is my first post in GitHub, sorry in advance if I missed some steps prior to this sending.
I am using mermaid V10.3 flow chart generator for an internal project. It generates IT architecture maps. Sometimes the maps are difficult to read as many arrows are going to a node. The node "ERP" is often too small to see properly the interfaces connected. In fact company's ERP often proposes many interfaces to other Applications.
We did not find the way to automatically increase the node size in relation with the amounts of arrow connected to it. Does Mermaid enable it ?
This will enable to increase the node "ERP" when they are many interfaces connected to it, and so we will be able to see for example the directions of the arrows...
Thanks a lot for your help.

**Comment by NathanDL** [comment]
I encountered the same situation. I programmatically padded my node titles to stretch them out horizontally. Could have done something similar with breaks for stretching vertically.
